packages/sources/react/src/components/forms/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -90,6 +90,9 @@ export const VtmnTextInput = ({
   onIconClick,
   ...props
 }: VtmnTextInputProps) => {
+  const propsToString = Object.keys(props).map((propKey) =>
+    props[propKey].toString(),
+  );
```

Comment:
Thanks! Do you think this could be more global for all components? (I think about making an High Order Component for example or a function that we use in all React components).

Reply:
I don't think a HOC would be a good solution in that case but a function would definitively do the trick.
(Function that will do the same thing, cast props to string)
But we would indeed need to edit every react component to use it.

Reply:
@wlalele okay for me, can you please add this fix for all components and create a function for this in a `utils` folder like we did in Svelte stack? Thanks a lot.?

Reply:
@lauthieb this bug fix is related to the text input component, I suggest you merge this one and I'll open a new PR for the other components as they require an in depth testing before merging.